cfme/networks/balancer.py

```diff
@@ -0,0 +1,95 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from navmazing import NavigateToSibling, NavigateToAttribute
+from utils.appliance import Navigatable
+from utils.update import Updateable
+from cfme.common import Taggable, SummaryMixin
+from cfme.networks.views import BalancerView
+from cfme.networks.views import BalancerDetailsView
+
+
+class BalancerCollection(Navigatable):
+    ''' Collection object for Balancer object
+    '''
+
+    def instantiate(self, name):
+        return Balancer(name=name)
+
+    def all(self):
+        view = navigate_to(Balancer, 'All')
+        list_networks_obj = view.entities.get_all(surf_pages=True)
+        return [Balancer(name=b.name) for b in list_networks_obj]
+
+
+class Balancer(Taggable, Updateable, SummaryMixin, Navigatable):
+    ''' Class representing balancers in sdn '''
+    in_version = ('5.8', version.LATEST)
+    category = "networks"
+    page_name = 'network_balancer'
+    string_name = 'NetworkBalancer'
+    refresh_text = "Refresh items and relationships"
+    detail_page_suffix = 'network_balancer_detail'
+    quad_name = None
+    db_types = ["NetworkBalancer"]
+
+    def __init__(
+            self, name, provider=None):
```

Comment:
**REQUIRED**
Please don't break the line. Also here and further where collections are used, `__init__()` must accept `appliance` and `collection` arguments.
```python
def __init__(self, name, provider=None, collection=None, appliance=None)
    if collection is None:
        collection = BalancerCollection(appliance=appliance)
    self.collection = collection
    Navigatable.__init__(self, appliance=collection.appliance)
    ...
```